build/cmake_cache.txt

```diff
@@ -0,0 +1,3 @@
+# Cross-compilation does strange things with CMake. Tell it the pthreads check passes.
+set(THREADS_PTHREAD_ARG "0" CACHE STRING "Result from TRY_RUN" FORCE)
```

Comment:
By my read of https://github.com/Kitware/CMake/blob/v3.9.0/Modules/FindThreads.cmake, don't you want to set this to "2", not "0"?

Reply:
yeah, I've tried a number of things, including setting it in the libroach cmake only. The only thing that's worked so far is setting it as a cache preset using `-C`. I honestly don't want to play with this crap again. It's broken, and this fixes it.

Reply:
ugh, now it works. Let's see if TC agrees.